tutorials/applications/Spin chain of 3 atoms in XY mode.ipynb

```diff
@@ -34,7 +34,7 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #12.    masked_pulse = Pulse.ConstantPulse(110, 2*np.pi*4.6, 0, 0)
</pre><p>This is quite obscure to be honest, why not use a BlackmanWaveform with an area of π instead?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pasqal-io/Pulser/pull/258/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>